test/worlds/hydrodynamics_flags.sdf

```diff
@@ -1,110 +1,108 @@
-<?xml version="1.0" ?>
-
+<?xml version="1.0"?>
 <sdf version="1.6">
   <world name="multi_lrauv">
     <physics name="1ms" type="ode">
       <max_step_size>0.001</max_step_size>
       <real_time_factor>1.0</real_time_factor>
     </physics>
-    <plugin
-      filename="ignition-gazebo-physics-system"
-      name="ignition::gazebo::systems::Physics">
-    </plugin>
-    <plugin
-      filename="ignition-gazebo-contact-system"
-      name="ignition::gazebo::systems::Contact">
+    <plugin filename="ignition-gazebo-physics-system"
+            name="ignition::gazebo::systems::Physics">
     </plugin>
-
-    <plugin
-      filename="ignition-gazebo-buoyancy-system"
-      name="ignition::gazebo::systems::Buoyancy">
+    <plugin filename="ignition-gazebo-buoyancy-system"
```

Comment:
Rather than have buoyancy here we could simply disable gravity

Reply:
Can. Done in https://github.com/gazebosim/gz-sim/pull/1819/commits/9d24ededf6da6b52eafeff00d5ba26dbfb2608b8.